.clang-format

```diff
@@ -1,493 +0,0 @@
-# SPDX-License-Identifier: GPL-2.0
-#
-# clang-format configuration file. Intended for clang-format >= 4.
-#
-# For more information, see:
-#
-#   Documentation/process/clang-format.rst
-#   https://clang.llvm.org/docs/ClangFormat.html
-#   https://clang.llvm.org/docs/ClangFormatStyleOptions.html
-#
```

Comment:
Based on https://github.com/angular/clang-format/blob/master/LICENSE, this .clang-format file is supposed to be under Apache 2.0 license. @er1cthe0ne do you still remember why GPL 2.0 is here? copy paste issue or?

Reply:
this .clang-format file could possibly be derived from here with its license header: https://github.com/CentaurusInfra/mizar/blob/dev-next/.clang-format

Reply:
Okay, the clang project is under Apache 2.0 after version 9.0. So it should be fine to use Apache 2.0 statement here.